Firmware/MVP/battsci.cpp

```diff
@@ -172,10 +199,11 @@ uint8_t BATTSCI_calculateChargeRequestByte(void)
 //Adjust final SoC value as needed to improve driving characteristics
 uint16_t BATTSCI_SoC_Hysteresis(uint16_t SoC_mappedToMCM_deciPercent)
 {
-  //TODO_NATALYA: Try to contain your SoC hysteresis variables and logic here.  If they need to be elsewhere, that's fine, but much cleaner to place here.
-  //examples:
-  //static uint16_t lastSoC_sentToMCM = 0;
-  //static uint16_t SoC_sentToMCMDelayIncrement...
+  if (SoC_mappedToMCM_deciPercent > previousOutputSoC_deciPercent) {
```

Comment:
Correction: I was wrong about this... while LiBCM's loop period is 10 milliseconds, since BATTSCI is only sent every 100 milliseconds, that means SoC can change at most 1% per second.  That is a 'good' rate of change, so this isn't required.